app/scripts/views/complete_reset_password.js

```diff
@@ -169,7 +169,7 @@ define(function (require, exports, module) {
     PasswordMixin,
     PasswordResetMixin,
     PasswordStrengthMixin,
-    ResendMixin,
+    ResendMixin(),
```

Comment:
Good point. An alternative is to invoke the function [where the module is required](https://github.com/mozilla/fxa-content-server/pull/4761/files#diff-10f1d562f7d1fbb313ab0b7219f873c1R17). 

```js
const ResendMixin = require('views/mixins/resend-mixin');
// changes to
const ResendMixin = require('views/mixins/resend-mixin')();
```